master/internal/resourcemanagers/kubernetes_resource_manager.go

```diff
@@ -97,6 +107,37 @@ func (k *kubernetesResourceManager) Receive(ctx *actor.Context) error {
 	return nil
 }
 
+func (k *kubernetesResourceManager) summarizeDummyResourcePool(
+	ctx *actor.Context,
+) *resourcepoolv1.ResourcePool {
+	slotsUsed := 0
+	for _, slotsUsedByGroup := range k.slotsUsedPerGroup {
+		slotsUsed += slotsUsedByGroup
+	}
+	return &resourcepoolv1.ResourcePool{
+		Name:                         kubernetesDummyResourcePool,
+		Description:                  "Kubernetes-managed pool of resources",
+		Type:                         resourcepoolv1.ResourcePoolType_RESOURCE_POOL_TYPE_K8S,
+		NumAgents:                    1,
+		SlotsAvailable:               int32(k.agent.numSlots()),
+		SlotsUsed:                    int32(k.agent.numUsedSlots()),
+		CpuContainerCapacity:         int32(k.agent.maxZeroSlotContainers),
+		CpuContainersRunning:         int32(k.agent.numZeroSlotContainers()),
+		DefaultGpuPool:               true,
+		DefaultCpuPool:               true,
+		Preemptible:                  false,
+		MinAgents:                    0,
+		MaxAgents:                    0,
+		CpuContainerCapacityPerAgent: 0,
+		SchedulerType:                resourcepoolv1.SchedulerType_SCHEDULER_TYPE_KUBERNETES,
+		SchedulerFittingPolicy:       resourcepoolv1.FittingPolicy_FITTING_POLICY_KUBERNETES,
+		Location:                     "kubernetes",
+		ImageId:                      "N/A",
```

Comment:
suggestion: leave empty or better leave out?

Reply:
I wanted to differentiate between cases where the ImageId is blank and is autofilled and the k8s case where the ImageId is a nonsensical field. I'm not super attached to this if you think this string is a bad idea.

Reply:
> cases where the ImageId is blank and is autofilled

question: would the this API's response, in this case, be `"" `or `"my-auto-filled-image-id"`?

Reply:
Hmm, it looks like it would be `"my-auto-fulled-image-id"`, so we could remove this `N/A`